packages/tezos-p2p/tezos-p2p.7.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "tezos-tooling" { with-test & = version }
+  "dune" { >= "1.11" }
+  "tezos-p2p-services" { = version }
+  "alcotest-lwt" { with-test & = "0.8.5" }
+  "lwt" { < "4.3" }
```

Comment:
Due to vagaries of the opam solver, it would be a good idea to put a minimum bound on lwt as well to ensure an ancient version of Lwt isn't select due to the upper bound.

Reply:
Good to know, shall I do it here now? This implies a new round of CI :-/

(Here too we would like to remove the constraint in the end and become compatible with Lwt 5. It appears to be super tricky and, for now, the tests become fork-bombs! with Lwt5. We are in contact with Lwt devs on that...)

Reply:
Done as other changes were required